BUILD

```diff
@@ -50,6 +50,11 @@ rust_library(
     ],
 )
 
+rust_library(
```

Comment:
cxx-build now depends on cxxbridge-flags and I had to add this to the bazel build.
Curiously enough, the buck build was ok?

Please note, I have no bazel experience, so this might be wrong/incomplete/...